src/parsers/power_models_data.jl

```diff
@@ -583,13 +592,13 @@ function read_branch!(sys::System, data, bus_number_to_bus::Dict{Int, Bus}; kwar
         return
     end
 
-    _get_name = get(kwargs, :branch_name_formatter, _get_pm_dict_name)
+    _get_name = get(kwargs, :branch_name_formatter, _get_pm_branch_name)
```

Comment:
1. It seems problematic if we are inferring information from the name. We should just need them to be unique. There's probably a detail I'm missing.
2. This same function is used in many other places. Notably, `make_dcline`. Is this a problem anywhere else?

Reply:
Re 1. It is common to use names to refer to lines. In a large system, operators use names and numbers, for instance, see the print screen with real line names. 

![image](https://user-images.githubusercontent.com/16385323/99126033-364b9580-25ba-11eb-8b07-c6cc5867a403.png)

Re 2. true, DC Line needs the same change.